R/nse-inject.R

```diff
@@ -368,6 +368,9 @@ NULL
 #' string using the default name operation.
 #'
 #' @param x A string to interpolate with glue operators.
+#' @param env User environment where the interpolation data lives in
+#'   case you're wrapping `englue()` in another function.
+#' @inheritParams args_error_context
 #'
```

Comment:
You have a section in details like

> Before using `englue()` in a package

It might also be useful to mention that `env` should be passed through there if the usage is exposed to the user somehow. It also comes up when wrapping glue itself https://github.com/tidyverse/glue/issues/281

Reply:
oh i see

```
@section Wrapping `englue()`:
```

now. Never mind